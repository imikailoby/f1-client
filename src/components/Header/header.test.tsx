/**
 * @jest-environment jsdom
 */
import { render } from '@testing-library/react';
import { Header } from './';

describe('<Header />', () => {
  it('renders without crashing', () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
