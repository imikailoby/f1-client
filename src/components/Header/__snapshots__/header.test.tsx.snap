// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders without crashing 1`] = `
<div>
  <header
    class="header"
  >
    <img
      alt="f1 logo"
      loading="lazy"
      src="svg>
"
    />
  </header>
</div>
`;
